xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.h

```diff
@@ -83,7 +83,7 @@ class CDVDDemuxFFmpeg : public CDVDDemux
   CDVDDemuxFFmpeg();
   ~CDVDDemuxFFmpeg() override;
 
-  bool Open(std::shared_ptr<CDVDInputStream> pInput, bool streaminfo = true, bool fileinfo = false);
+  bool Open(std::shared_ptr<CDVDInputStream> pInput, bool fileinfo = false);
```

Comment:
I would eliminate the default for the fileinfo param too. Default parameters are pest. Open was allowed to be called with two parameters. Those calls (if they exist) will result in runtime errors now because not detected by the compiler.

Reply:
Good idea, done.